seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/task/SourceSeaTunnelTask.java

```diff
@@ -76,10 +81,17 @@ public void init() throws Exception {
                             + startFlowLifeCycle.getClass().getName());
         } else {
             SeaTunnelDataType sourceProducedType;
+            List<String> tableNames = new ArrayList<>();
             try {
                 List<CatalogTable> producedCatalogTables =
                         sourceFlow.getAction().getSource().getProducedCatalogTables();
                 sourceProducedType = CatalogTableUtil.convertToDataType(producedCatalogTables);
+                tableNames =
+                        producedCatalogTables.stream()
+                                .map(CatalogTable::getTableId)
+                                .map(TableIdentifier::getTableName)
+                                .filter(StringUtils::isNotBlank)
+                                .collect(Collectors.toList());
```

Comment:
ditto. Please use TablePath.
```suggestion
                tableNames =
                        producedCatalogTables.stream()
                                .map(CatalogTable::getTableId)
                                .map(TableIdentifier::toTablePath)
                                .collect(Collectors.toList());
```

Reply:
ok